lang/en/tool_cloudmetrics.php

```diff
@@ -85,4 +85,8 @@
 
 // Error messages.
 $string['metric_not_found'] = 'Metric \'{$a}\' not found';
-
+$string['collector_failed'] = 'Metric \'{$a}\' failed %s';
+$string['collector_passed'] = 'Metric \'{$a}\' succeeded';
+$string['no_issues'] = 'No issues';
+$string['checkcollectorcheck'] = 'Collector Failures';
```

Comment:
Also this label doesn't mean much out of context if you are looking at the status report. I think it could be more like 'Cloud metric collectors'